### PR TITLE
Implement readme.md

### PR DESCRIPTION
**Pull request by ddouglasz**
### What does this PR do?

- adds a clear Readme on the purpose of the project.
- provides a PR template

### Description of Task to be completed?

- provides users and collaborators with a context of the project direction

### How should this be manually tested?

- Navigate to README.md file to find readme.

### Any background context you want to provide?

- N/A

### Questions:

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply frontend/public/index.html]: Okay awesome.I would get all those done now. Oversight. We have a new Redo of the project.

[the remark this answers is not in the document]
Comment (Rhotimee) [reply frontend/public/index.html]: 👍🏽

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- navigate to the folder on command prompt or terminal.
+- create a `.env` file from the .env.example file.
+- create mongoDB database and add to the `.env` file
+- cd in to the backend.
+- run $ `npm run dev`
+- also cd into the frontend
+- run $ `npm run start`P
+
+## How do I contribute to this project
+
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## How do I set up on my local machine
+
+- Clone the app to your local machine
+- navigate to the folder on command prompt or terminal.
+- create a `.env` file from the .env.example file.
+- create mongoDB database and add to the `.env` file
+- cd in to the backend.
+- run $ `npm run dev`
+- also cd into the frontend
+- run $ `npm run start`P
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- MongoDB(Database)
+- GraphQL(frontend/backend)
+
+## How do I set up on my local machine
+
+- Clone the app to your local machine
+- navigate to the folder on command prompt or terminal.
+- create a `.env` file from the .env.example file.
+- create mongoDB database and add to the `.env` file
+- cd in to the backend.
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- Node.js(Backend)
+- MongoDB(Database)
+- GraphQL(frontend/backend)
+
+## How do I set up on my local machine
+
+- Clone the app to your local machine
+- navigate to the folder on command prompt or terminal.
+- create a `.env` file from the .env.example file.
+- create mongoDB database and add to the `.env` file
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+Born out of the growing need to solve the problem of children in need of community assistance to at least have some basic amenities,  the **iGive** connects users to the nearest orphanage where they would like to make donations and provide aids. 
+
+Imagine a situation where you were taking clothes to an orphanage whose immediate need was just some bunch of tissue papers, just because there was no platform to find updates on what some orphanges are in immediate need of. Igive solves that problem by providing you with the specific details of what each orphanage needs.
+
+
+ 
+# Technologies used:
+
+- React.js(Client)
+- Node.js(Backend)
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

File: README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+- MongoDB(Database)
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply README.md]: done

**Pull request merged**